src/framework-dfp.c

```diff
@@ -1164,6 +2810,8 @@ bool* innermost_dep_satisfaction_dims(PlutoProg *prog, bool *tile_preventing_dep
     return sat_dim;
 }
 
+/* Returns skewing constraints. The lowerbounds of the 
```

Comment:
lowerbounds -> lower bounds

Also, please use the width better (wrap maximally at 80 columns).